### PR TITLE
Handle title text on images

### LINKED ISSUE #8: Image title text ignored
Per the [Markdown syntax for images](https://daringfireball.net/projects/markdown/syntax#img), an optional title attribute can be provided following the url in single or double quotes. Example: `Alt text`

The processor seems to be taking the url, alt text, and a title as arguments but [the title arg is unused](https://github.com/gitterHQ/gitter-markdown-processor/blob/78a00600aef9642fb2154e1843fb175e780162d1/lib/process-chat.js#L190-L195).

Is there a reason for this?

### PR DESCRIPTION
**Pull request by keawade**
Resolves #8.

### CODE REVIEW

**Comment by MadLittleMods** [comment]
How does this handle image syntax without the title? It looks like it might `undefined` or empty string in the attributes which is undesired.

```md
![This is alt text](https://www.gitter.im/)
```

**Comment by keawade** [comment]
I'm not certain, but I suspect you are correct. I'll check that this evening and correct it as needed.

**Comment by keawade** [comment]
@MadLittleMods Okay, I've resolved the `null` title issue and added a test for that case.

File: lib/process-chat.js [hunk 1]
```diff
@@ -190,8 +190,11 @@ function getRenderer(renderContext) {
   renderer.image = function(href, title, text) {
     href = checkForIllegalUrl(href);
     renderContext.urls.push({ url: href });
-    return util.format('<img src="%s" alt="%s" rel="nofollow">', href, text);
-
+    if (title) {
+      return util.format('<img src="%s" title="%s" alt="%s" rel="nofollow">', href, title, text);
+    } else {
+      return util.format('<img src="%s" alt="%s" rel="nofollow">', href, text)
```
Comment (MadLittleMods) [inline lib/process-chat.js]: Missing semicolon

**Review by MadLittleMods — CHANGES_REQUESTED** [review]

**Comment by MadLittleMods** [comment]
@keawade Looks fine. Just a small nit 😀

**Comment by keawade** [comment]
Whoops, my bad. Fixed.

**Comment by MadLittleMods** [comment]
Thanks for the contribution @keawade 🎉

**Pull request merged**